ui/src/types/services.ts

```diff
@@ -4,8 +4,10 @@ export interface NewService {
   type: string
   username?: string
   password?: string
-  active: boolean
   insecureSkipVerify: boolean
+  metadata?: {
+    [x: string]: any
```

Comment:
this is a generic service as opposed to a flux service. A flux service could but does not necessarily have active in the metadata and a flux service is a service that has type 'flux'. we didnt want to make it specific to flux here since we treat it as though its is any service.